tests/blockchain/eth/contracts/integration/test_contract_economics.py

```diff
@@ -0,0 +1,65 @@
+"""
+This file is part of nucypher.
+
+nucypher is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+nucypher is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+import pytest
+
+
+# Experimental max error
+MAX_ERROR = 0.0004751
+MAX_PERIODS = 100
+
+
+@pytest.mark.slow
+def test_reward(testerchain, three_agents, token_economics):
+    testerchain.time_travel(hours=1)
+    token_agent, miner_agent, _policy_agent = three_agents
+    origin = testerchain.etherbase_account
+    ursula = testerchain.ursula_account(0)
+
+    # Prepare one staker
+    _txhash = token_agent.transfer(amount=token_economics.minimum_allowed_locked,
```

Comment:
In principle, if you're not using `_txhash` anyhow, you can omit `_txhash = ` part for the sake of readability

Reply:
I'm just copied style from other tests. So maybe @KPrasch will help me